plugins/catalog/src/components/HasComponentsCard/HasComponentsCard.tsx

```diff
@@ -33,6 +33,14 @@ type Props = {
   variant?: 'gridItem';
 };
 
+const columns = [
+  EntityTable.columns.createEntityRefColumn({ defaultKind: 'component' }),
```

Comment:
These are case insensitive right? It was all upper API above, just checking

Reply:
Yeah, I think it should be: https://github.com/backstage/backstage/blob/6c09312dcab36ea7d2e820aa5d1af65501b23c3e/plugins/catalog-react/src/components/EntityRefLink/format.ts#L50